tests/PHPStan/Analyser/data/ignore-next-line.php

```diff
@@ -17,16 +17,6 @@ public function doFoo(): void
 
 		/** @phpstan-ignore-next-line */
 		fail();
-
-		/*
-		 * @phpstan-ignore-next-line
-		 */
-		fail();
-
-		/**
-		 * @phpstan-ignore-next-line
```

Comment:
This is tricky BC break, in my POV, this was always wrong since it was not ignoring next line. It could even be N + 10 line.

```
/*
 * @phpstan-ignore-next-line
 * @param $notIgnored
 * @throws StillNotIgnored
 */
ignored();
```

But it will most probably break stuff in some codebases, so we may use this fixed behaviour only for bleedingEdge.

Reply:
Yeah, it occurs from time to time: https://github.com/search?q=%40phpstan-ignore-next-line&type=code

Reply:
So I kept the old behaviour for non bleeding edge.